PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
Why this check, then? A comment _at the very least_ as to its purpose is desirable.

I understand that, generally, `$` is jQuery in JavaScript. My _guess_ is that this is a hyper-conservative way to ensure that no one "renamed" the ExtendScript `$` Helper object and didn't somehow replace it with something else. _If_ this is the case, then some comments about this fact would be very helpful for those trying to piece together how the environment actually works.

The same holds true for [PProPanel.jsx](https://github.com/mBeierl/Samples/blob/d53d277874b951eb57d46ffee6a658748663ebf5/PProPanel/PProPanel.jsx), actually. `$` is a super important object in ExtendScript contexts and to the uninitiated, that code simply looks like you might be grafting something onto jQuery or another library...